Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerHeadlessUniversalCheckout.swift

```diff
@@ -39,7 +47,28 @@ public class PrimerHeadlessUniversalCheckout {
             print("WARNING!\nPrimerHeadlessUniversalCheckout delegate has not been set, and you won't be able to receive the Payment Method Token data to create a payment.")
         }
         
-        DependencyContainer.register(settings ?? PrimerSettings() as PrimerSettingsProtocol)
+        PrimerInternal.shared.sdkIntegrationType = .headless
+        PrimerInternal.shared.intent = .checkout
+        
+        self.checkoutSessionId = UUID().uuidString
+        self.timingEventId = UUID().uuidString
+        
+        let sdkEvent = Analytics.Event(
+            eventType: .sdkEvent,
+            properties: SDKEventProperties(
+                name: #function,
+                params: [
+                    "intent": PrimerInternal.shared.intent?.rawValue ?? "null"
+                ]))
+        
+        let timingStartEvent = Analytics.Event(
+            eventType: .timerEvent,
+            properties: TimerEventProperties(
+                momentType: .start,
+                id: self.timingEventId))
+        
+        Analytics.Service.record(events: [sdkEvent, timingStartEvent])
+        Analytics.Service.sync()
```

Comment:
Why do you need to call `sync` all the time?

Reply:
No need actually. Removed them.